website/docs/reference/dbt-jinja-functions/adapter.md

```diff
@@ -106,6 +107,30 @@ Returns a [Relation](dbt-classes#relation) object identified by the `database.sc
 
 </File>
 
+## load_relation
+__Args__:
+
+ * `relation`: The database of the relation to fetch
```

Comment:
Is this a correct description?

Or is the following more accurate?
`relation`: The relation to find within the database

Reply:
Or maybe this:
`relation`: The relation to attempt looking up via the adapter